game/resource/English/ability/items/tooltip_shivas_cuirass.txt

```diff
@@ -37,3 +37,4 @@
             "DOTA_Tooltip_Ability_item_shivas_cuirass_2_blast_debuff_duration"              "BLAST SLOW DURATION:"
             "DOTA_Tooltip_Ability_item_shivas_cuirass_2_blast_radius"                       "BLAST RADIUS:"
 
+            "dota_tooltip_modifier_item_shivas_cuirass_negative"                            "Shiva's Curse"
```

Comment:
IIRC from looking at the Shiva's Cuirass code, the positive and negative aren't actually used. Probably set it up, changed to a single modifier cause it was simpler to implement then forgot to remove them.